### ISSUE TITLE
cy.type() into email input gives error

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#504]
* Operating System: OSX
* Cypress Version: 0.19.2

### Are you requesting a feature or reporting a bug?
Bug

### Current behavior:
The Kitchen Sink currently fails on `cy.type()` into email input with this error:

```
Error: Failed to execute 'setSelectionRange' on 'HTMLInputElement': 
The input element's type ('email') does not support selection.
```

**Suspect it is related to Chrome Version 58.0.3029.110**

#### Test code:

*Kitchen Sink Example*
```js
cy.get('.action-email').type('fake@email.com').clear()
```

```js
cy.type('{leftarrow}{rightarrow}{uparrow}{downarrow}{del}{selectall}{backspace}')
```

### Additional Info (images, notes, stack traces, etc)

<img width="451" alt="screen shot 2017-05-18 at 9 50 03 am" src="https://cloud.githubusercontent.com/assets/1271364/26205754/b8ec03aa-3bb0-11e7-94af-b2bc548ca271.png">

**Comment by ddeath** [comment]
The same is happening to me on Google Chrome Version 60.0.3112.7 (Official Build) dev (64-bit) on Electron the test are working fine

![cyp_bug](https://cloud.githubusercontent.com/assets/2537379/26642564/7df9e5ea-462e-11e7-9066-61df786031ef.png)

**Comment by brian-mann** [comment]
This is fixed in our next release `0.20.0`

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
